modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf

```diff
@@ -14,18 +14,12 @@
  */
 --%>
 
+<% boolean isShowingExpirationError = false; %>
```

Comment:
We can bring this declaration closer to its use (just before the `<liferay-ui:error exception="<%= DDMFormValuesValidationException.MustSetValidValue.class %>">`) and what do you think about renaming it to `showExpirationDateError`?

Reply:
That's great! I will try that!